packages/babel/src/transform-stages/helpers/loadLinariaOptions.ts

```diff
@@ -37,13 +39,13 @@ export default function loadLinariaOptions(
       },
       {
         // The old `ignore` option is used as a default value for `ignore` rule.
-        test: ignore ?? /[\\/]node_modules[\\/]/,
+        test: ignore ?? nodeModulesRegExp,
         action: 'ignore',
       },
       {
         // Do not ignore ES-modules
         test: (filename, code) => {
-          if (!/\/node_modules\//.test(filename)) {
+          if (!nodeModulesRegExp.test(filename)) {
```

Comment:
Shouldn't the `!` be removed? Don't we want to ignore node modules?
```suggestion
          if (nodeModulesRegExp.test(filename)) {
```

Reply:
No, it shouldn't.  We can't just ignore node_modules because we have to process libraries which were transpiled esm, so we made a dirty check here: if a processed file in node_modules and has something that looks like an import or export statement, we will not ignore it.